app/src/components/governance/PageProposals.vue

```diff
@@ -125,7 +177,17 @@ export default {
   }),
   computed: {
     ...mapState([`connection`]),
-    ...mapGetters([`network`, `currentNetwork`]),
+    ...mapGetters([`currentNetwork`, `stakingDenom`]),
+    fundTitle() {
```

Comment:
Let's find one name and not use 2?

Reply:
oops, I saw an error there, it should be `this.currentNetwork.network_type === "polkadot"` , not `this.currentNetwork.id === "polkadot"`